internal/livelint/interaction.go

```diff
@@ -0,0 +1,60 @@
+package livelint
+
+import (
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+//go:generate moq -out mock_ui_test.go -pkg livelint_test . UserInteraction
+
+type UserInteraction interface {
+	DisplayVerboseSetting(verbose bool)
```

Comment:
IMO the verbose flag should not be something that is displayed, but a setting in the UI config. The code can then send all the messages it could possibly send and the output of additional details in verbose mode is then an implementation detail of the ui.

See also the comment on where this function is called.

Reply:
Agreed. Removed that flag from the UI.